roles/install/tasks/yum/prepare.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Include distro-specific vars ({{ ansible_distribution }})
+  include_vars: file='{{ ansible_distribution }}.yml'
+
+- name: Add yum repository
+  yum_repository:
+    name: sensu_stable
+    description: sensu_stable
+    file: sensu
+    baseurl: https://packagecloud.io/sensu/stable/{{ os }}/{{ dist }}/$basearch
+    gpgkey: https://packagecloud.io/sensu/stable/gpgkey
+    gpgcheck: no
+    repo_gpgcheck: yes
+    enabled: yes
+    sslverify: yes
+    sslcacert: /etc/pki/tls/certs/ca-bundle.crt
```

Comment:
Is this line needed? I would assume that root certs are taken from here by default. But then again, I might be completely wrong.